Makefile

```diff
@@ -1,9 +1,15 @@
 SOURCE_FILES := $(shell test -e src/ && find src -type f)
+VERSION := $(shell sed --posix -n 's,^version = \"\(.*\)\",\1,p' Cargo.toml)
 
 policy.wasm: $(SOURCE_FILES) Cargo.*
 	cargo build --target=wasm32-wasi --release
 	cp target/wasm32-wasi/release/*.wasm policy.wasm
 
+artifacthub-pkg.yml: policy.wasm metadata.yml
```

Comment:
The `artifacthub-pkg.yml` target should depend on the files `metadata.yml Cargo.toml`.

We are depending incorrectly on the file `policy.wasm`, Hence make will trigger the call to the target `policy.wasm`, which rebuilds the policy before recreating the `artifacthub-pkg.yml`

The `check-artifacthub` CI job, doesn't have cargo properly setup, so hence why the job fails so spectacularly:
https://github.com/kubewarden/allowed-proc-mount-types-psp-policy/actions/runs/4471275469/jobs/7856046403#step:13:24018